src/js/shield_defs.js

```diff
@@ -69,15 +69,15 @@ function roundedRectShield(
   textColor = textColor ?? strokeColor;
   radius = radius ?? 2;
   return {
-    backgroundDraw: (ref) =>
-      ShieldDraw.roundedRectangle(
-        fillColor,
-        strokeColor,
-        ref,
-        radius,
-        1,
-        rectWidth
```

Comment:
@claysmalley this is where the original code defaulted to a 1px radius, so I just moved that downstream.

Reply:
Ha, all the more reason why the options object is cleaner.  Done.